cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -756,6 +762,9 @@ printPlan verbosity
     showMonitorChangedReason  MonitorFirstRun     = "first run"
     showMonitorChangedReason  MonitorCorruptCache = "cannot read state cache"
 
+    showBuildProfile = "Build profile:\n" ++ (unlines [
```

Comment:
Please try to squeeze this into a single line; this is mostly redundant information to see on every invocation, so it better be terse/compact